staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -5094,7 +5093,6 @@ func printLabelsMultilineWithIndent(w PrefixWriter, initialIndent, title, innerI
 			w.Write(LEVEL_0, "%s", innerIndent)
 		}
 		w.Write(LEVEL_0, "%s=%s\n", key, labels[key])
-		i++
```

Comment:
Does this mean we have been mis-printing since forever? It doesn't seem to be in trivial tests..

Reply:
I don't see that `i` is being used in another place that the first `if`,  so it works because it only cares about when it is 0